code/modules/mob/living/brain/brain_item.dm

```diff
@@ -147,6 +147,23 @@
 	icon_state = "brain-x"
 	origin_tech = "biotech=6"
 
+/obj/item/organ/brain/cybernetic/vox
+	name = "Vox cortical stack"
+	slot = "brain"
+	desc = "A vox cortical stack - considered to be their brain. Its existance is the center of debate."
```

Comment:
a) existence*
b) what does that even mean? People can see it pretty clearly, right? What's there to debate?

Reply:
A joke about vox not having brains, mostly.